apps/hannk/delegate/hannk_delegate.cpp

```diff
@@ -1396,9 +1404,6 @@ class NodeSupport {
         if (!IsVersionOK(1, 2)) {
             return false;
         }
-        if (!InputsHaveCorrectTypes({ANY})) {
```

Comment:
Removing this means that the shape op can have any number of inputs; this call ensures that there is exactly one input (but it can be of any type). Is that what we want?

Reply:
Good point, fixed